cmd/switch.go

```diff
@@ -33,11 +33,13 @@ func newSwitchCmd(homedir string) *cobra.Command {
 	var flag switchFlags
 
 	cmd := &cobra.Command{
-		Use:   "switch <istio version>",
+		Use:   "switch <istio version | flavor | flavor-version>|<istio version full name>",
```

Comment:
I don't think <..> is not necessary. See https://github.com/tetratelabs/getistio/blob/main/cmd/fetch.go#L35
```suggestion
		Use:   "switch",
```